From 29c02401a177b481d0a6f4ccee4af925c82e8c30 Mon Sep 17 00:00:00 2001
From: Daniel Almeida <daniel.almeida@collabora.com>
Date: Mon, 17 Nov 2025 09:53:17 -0300
Subject: [PATCH 153/161] tyr: job: don't check 'latest_flush'

This check needs to be reworked. For now, remove it.
---
 drivers/gpu/drm/tyr/sched/job.rs | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/tyr/sched/job.rs b/drivers/gpu/drm/tyr/sched/job.rs
index 6a16fa514d..ddb0353c37 100644
--- a/drivers/gpu/drm/tyr/sched/job.rs
+++ b/drivers/gpu/drm/tyr/sched/job.rs
@@ -62,11 +62,6 @@ pub(crate) fn create(
             return Err(EINVAL);
         }
 
-        if qsubmit.latest_flush & genmask_u32(24..=30) != 0 {
-            pr_err!("job_create: latest_flust[30:24] must be zero\n");
-            return Err(EINVAL);
-        }
-
         Ok(Job {
             group: group.clone(),
             queue_idx: qsubmit.queue_index as usize,
-- 
2.51.0

